solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -139,13 +139,8 @@ contract LiquidityProtection is ILiquidityProtection, TokenHandler, Utils, Owned
 
     // ensures that the contract is currently removing liquidity from a converter
     modifier updatingLiquidityOnly() {
-        _updatingLiquidityOnly();
-        _;
-    }
-
-    // error message binary size optimization
-    function _updatingLiquidityOnly() internal view {
         require(updatingLiquidity, "ERR_NOT_UPDATING_LIQUIDITY");
```

Comment:
is this used only once? otherwise it would increase binary size

Reply:
Yes, it is used once.